src/libs/pwa.js

```diff
@@ -1,81 +0,0 @@
-/* eslint-disable no-unused-vars */
```

Comment:
Good call on removing this code for now - they are some random notes that I added, to be integrated in when we re-enable the service worker functionality/add in offline fallbacks. I will add this code as a comment to the service worker ticket.